### DISCUSSION TITLE
Fox Family Movies

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#722] [Q&A]
**Channel Name:** Fox Family Movies
**Channel ID:** FoxFamilyMovies.us
**Country:** Middle East
**Language:** English
**Website:** xxx
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by freearhey** [comment]
According to our database this channel was closed on 1 October 2021.

https://iptv-org.github.io/channels/us/FoxFamilyMovies
